helloworld-jersey/build.gradle

```diff
@@ -16,8 +16,12 @@ repositories {
 }
 
 dependencies {
-    compile(project(':integration-enabler-jersey'))
+    compile(project(':integration-enabler-java'))
 
+    compile libraries.springFox
+    compile group: 'io.swagger', name: 'swagger-jersey2-jaxrs', version: '1.5.10'
```

Comment:
Hello Andrea, I am doing more testing. It looks great! @JirkaAichler would say that you should use the same style as for the other dependencies: `compile libraries.` in this and other `build.gradle` files

Reply:
Hi Petr, yes I should use the same style! I'll be changing it!

Reply:
Done :)